package.json

```diff
@@ -37,21 +37,17 @@
     "fs-extra": "^9.0.0",
     "mocha": "^7.1.1",
     "nyc": "^15.0.0",
-    "prebuildify": "^4.0.0",
-    "prebuildify-ci": "^1.0.5",
     "ref-array-di": "^1.2.1"
   },
   "scripts": {
     "install": "node-gyp-build",
```

Comment:
Hi @tarmack, removing prebuild script here results in broken build for appveyor over Node v12 since there is the following condition if ($env:nodejs_version -eq "12") then npm run prebuild.

Are you planning to fix this on your PR ?

Regards,
Antony